configs/components/cpp-pcp-client.json

```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/cpp-pcp-client.git", "ref": "refs/tags/1.1.2"}
+{"url": "git://github.com/puppetlabs/cpp-pcp-client.git", "ref": "master"}
```

Comment:
Still refs/tags/1.1.2. We haven't done a 1.1.3 release, I plan to do that today/tomorrow.